rapids/test_rapids.py

```diff
@@ -47,43 +58,53 @@ def verify_spark_job(self):
           "{}/rapids/{}".format(self.INIT_ACTIONS_REPO,
                                 self.XGBOOST_SPARK_TEST_SCRIPT_FILE_NAME))
 
-  @parameterized.parameters(("STANDARD", ["m", "w-0"], GPU_P100))
-  def test_rapids_dask(self, configuration, machine_suffixes, accelerator):
-    if self.getImageVersion() < pkg_resources.parse_version("1.5"):
-      self.skipTest("Not supported in pre 1.5 images")
-
-    optional_components = None
+  @parameterized.parameters(
+    ("STANDARD", ["m", "w-0"], GPU_P100, None),
+    ("STANDARD", ["m", "w-0"], GPU_P100, "yarn"),
+    ("STANDARD", ["m"], GPU_P100, "standalone"))
+  def test_rapids_dask(self, configuration, machine_suffixes, accelerator, dask_runtime):
     if self.getImageVersion() < pkg_resources.parse_version("2.0"):
-      optional_components = ["ANACONDA"]
+      self.skipTest("Not supported in pre 2.0 images")
+
+    init_actions = self.INIT_ACTIONS[:]
+    init_actions.insert(1, "dask/dask.sh")
+
+    metadata="gpu-driver-provider=NVIDIA,rapids-runtime=DASK"
+    if dask_runtime:
+      metadata+=",dask-runtime={}".format(dask_runtime)
 
     self.createCluster(
         configuration,
-        self.INIT_ACTIONS,
-        metadata="gpu-driver-provider=NVIDIA,rapids-runtime=DASK",
-        machine_type="n1-standard-2",
+        init_actions,
+        metadata=metadata,
+        machine_type="n1-standard-8",
         master_accelerator=accelerator,
         worker_accelerator=accelerator,
-        optional_components=optional_components,
-        timeout_in_minutes=60)
+        timeout_in_minutes=30)
 
     for machine_suffix in machine_suffixes:
       self.verify_dask_instance("{}-{}".format(self.getClusterName(),
-                                               machine_suffix))
+                                               machine_suffix),
+                                dask_runtime)
 
   @parameterized.parameters(("STANDARD", ["w-0"], GPU_P100))
   def test_rapids_spark(self, configuration, machine_suffixes, accelerator):
     if self.getImageVersion() < pkg_resources.parse_version("1.5"):
       self.skipTest("Not supported in pre 1.5 images")
 
+    optional_components = None
+
     metadata = "gpu-driver-provider=NVIDIA,rapids-runtime=SPARK"
     if self.getImageVersion() < pkg_resources.parse_version("2.0"):
       metadata += ",cuda-version=10.1"
+      optional_components = ["ANACONDA"]
 
     self.createCluster(
         configuration,
         self.INIT_ACTIONS,
+        optional_components=optional_components,
         metadata=metadata,
-        machine_type="n1-standard-2",
+        machine_type="n1-standard-8",
```

Comment:
Do we really need so big VMs? Will `n1-standard-4` work?

Reply:
Ack, I'll try